src/Symfony/Component/Form/Console/Descriptor/Descriptor.php

```diff
@@ -113,6 +102,27 @@ protected function collectOptions(ResolvedFormTypeInterface $type)
         $this->extensions = array_keys($this->extensions);
     }
 
+    protected function getOptionDefinition(OptionsResolver $optionsResolver, $option)
+    {
+        $refObject = new \ReflectionObject($optionsResolver);
+        foreach (array('defaults', 'lazy', 'allowedTypes', 'allowedValues', 'normalizers') as $name) {
+            $property = $refObject->getProperty($name);
+            $property->setAccessible(true);
+            $value = $property->getValue($optionsResolver);
+            if (array_key_exists($option, $value)) {
+                $definition[$name] = $value[$option];
+            }
+        }
+        $definition['required'] = $optionsResolver->isRequired($option);
+
+        if (isset($definition['lazy'])) {
+            $definition['defaults'] = 1 === count($definition['lazy']) ? $definition['lazy'][0] : $definition['lazy'];
```

Comment:
shouldnt default value be expanded? This will dump a closure no?

Reply:
Yes, can dump a \Closure as at this point the option has not been resolved.